src/libs/routes/validationHandler.ts

```diff
@@ -0,0 +1,81 @@
+export default config => (req, res, next) => {
+  console.log("Inside Middleware");
+  const keys = Object.keys(config);
+
+  keys.forEach(key => {
+    const item = config[key];
+    console.log("Items are ", item);
+    let values = item.in.map(item => {
+      console.log("REQUEST---", req[item][key]);
+      return req[item][key];
+    });
+    console.log("Values are", values);
+    if (item && item.required) {
+      const validatedValues = values.filter(item => item);
+      if (validatedValues.length != values.length) {
+        console.log("ERROR----");
+        next({
+          status: "Bad Request",
+          message: item.errorMessage || `${key} is required` || "Error Message"
+        });
+      }
+      Object.keys(values).map(function(key) {
```

Comment:
Please check for string is not for all keys.Please wrap that inside item.string check.